.github/workflows/tests.yaml

```diff
@@ -31,36 +20,44 @@ jobs:
         done;
 
     - name: load test GET endpoint
-      run: |
-        set -e
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/get
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/get
+        EOF
+    - uses: iter8-tools/iter8-action@v1
+      with:
+        chart: load-test-http
+        valuesFile: experiment-config.yaml
 
     - name: load test GET endpoint
-      run: |
-        set -e
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/get \
-                  --set SLOs.http/latency-p25=80 \
-                  --set SLOs.http/latency-p50=100 \
-                  --set SLOs.http/latency-'p95\.0'=200
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/get
+          SLOs:
+            http/latency-p25: 80
+            http/latency-p50: 100
+            http/latency-p95.0: 200
+        EOF
+    - uses: iter8-tools/iter8-action@v1
+      with:
+        chart: load-test-http
+        valuesFile: experiment-config.yaml
 
-    - name: load test POST endpoint (image/jpeg)
-      run: |
-        set -e
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/post \
-                  --set payloadURL=https://cdn.pixabay.com/photo/2021/09/08/17/58/poppy-6607526_1280.jpg \
-                  --set contentType="image/jpeg" \
-                  --set SLOs.http/latency-p25=80 \
-                  --set SLOs.http/latency-p50=100 \
-                  --set SLOs.http/latency-'p95\.0'=200
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+    - name: load test POST endpoint
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/post
```

Comment:
Host information should be set in the same manner as gRPC -- extracting the address from docker bridge seems like the more robust approach